src/app/_components/RecentBlocks/utils.ts

```diff
@@ -0,0 +1,16 @@
+export function formatTimestamp(timestamp: number) {
```

Comment:
I have a very similar function for the tables. I think we should try to consolidate util files. They are starting to get out of hand because there are so many of them

Reply:
totally agree, we should bring it up in next meeting